Targets/JavaScriptCore/REVISION

```diff
@@ -1 +1 @@
-85041c84e3ed873d6a9d77e276f08eed41dfa648
+3da169520a4cc62ff0ea01e8cc36a8d6a15a809f
```

Comment:
Do the previous patches apply cleanly (i.e. without offsets etc.) to the new revision? Otherwise, please also include the new webkit.patch :)